lib/Controller/Display.php

```diff
@@ -500,7 +505,10 @@ function grid()
             /* @var \Xibo\Entity\Display $display */
             if (in_array('displaygroups', $embed)) {
                 $display->load();
-            } else {
+            } elseif (in_array('overrideconfig', $embed)) {
```

Comment:
shouldn't this be a new IF statement after this one - what if you want to include displaygroups and overrideconfig?